nss-day-cohort-43-csharp/Fullstack-Rage-Against-The-Virtual-Machine
# Rb users reactivate

Issue [28](https://github.com/nss-day-cohort-43-csharp/Fullstack-Rage-Against-The-Virtual-Machine/issues/28)

Allows an admin user to reactivate deactivated accounts

Testing:
- Log in as an admin
- Navigate to the user profiles page
- When you click the "Deactivated" button below the header, a list of deactivated accounts should be shown
- When you go to activate an account, check to see both the "No, cancel" and "Yes, activate" buttons work as expected
- When an account has been reactivated, if should have all permissions restored and should be able to navigate the site
